views/jira-configuration-new.hbs

```diff
@@ -214,28 +214,31 @@
                           >
                             <span class="jiraConfiguration__appDropdownIcon">...</span>
                           </button>
-                          <aui-dropdown-menu class="" id="app-settings-dropdown-{{id}}">
-                            <aui-section>
-                              <!--TODO: need to add url for this-->
-                              <a
-                                  class=""
-                                  href="#"
-                                  data-installation-link="{{html_url}}"
-                                  target="_blank"
-                              >
-                                Edit
-                              </a>
-                              <!--TODO: need to add functionality for this-->
-                              <a
-                                  class=""
-                                  href="#"
-                                  data-installation-link="{{html_url}}"
-                                  target="_blank"
-                              >
-                                Delete
-                              </a>
-                            </aui-section>
-                          </aui-dropdown-menu>
+                          <aui-dropdown-menu class="jiraConfiguration__table__dropdown" id="app-settings-dropdown-{{id}}">
+                          <h6 class="jiraConfiguration__table__cell__settings__header">
+                            <span class="jiraConfiguration__table__cell__settings__headerItem">Application</span>
+                          </h6>
+                          <aui-section class="jiraConfiguration__table__cell__settings__dropdownItems">
+                             <div class="jiraConfiguration__table__cell__gitHubAppItem">
+                               <button
```

Comment:
Not sure why we were using links here because we're not navigating outside the app 🤷 Had to wrap each list item in a div as well otherwise clicking on one would fire the other (both at the same time).